google-cloud-spanner-hibernate-samples/basic-hibernate-sample/src/main/java/com/example/Payment.java

```diff
@@ -40,7 +41,7 @@
   @Type(type = "uuid-char")
   private UUID id;
 
-  private Long amount;
+  private BigDecimal amount;
```

Comment:
Are we still testing `Long` somewhere else?

Reply:
No worries, we have plenty of `Long` coverage (among other data types) in the Hibernate integration tests that we copied from the Hibernate repo: https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/blob/master/google-cloud-spanner-hibernate-testing/src/main/java/org/hibernate/userguide/model/Payment.java#L28